scripts/openwrt_upload.sh

```diff
@@ -2,6 +2,4 @@
 set -eux
 bash scripts/openwrt_build.sh
 scp target/mipsel-unknown-linux-musl/debug/rita root@192.168.1.1:/tmp/rita
-scp settings/default.toml root@192.168.1.1:/etc/rita-default.toml
-scp scripts/rita-test.toml root@192.168.1.1:/etc/rita.toml
-ssh root@192.168.1.1 RUST_BACKTRACE=FULL RUST_LOG=TRACE /tmp/rita --config /etc/rita.toml  --default /etc/rita-default.toml --platform openwrt
+ssh root@192.168.1.1 RUST_BACKTRACE=FULL RUST_LOG=TRACE /tmp/rita --config /etc/rita.toml --platform linux &> out.log
```

Comment:
Wouldn't it be better to keep the IP in an env? Also, is `/tmp/` guaranteed to be in-RAM on all platforms? Personally I like using `scp` for this on `/usr/sbin/rita` itself cause it'll substitute the binary in-place.

Reply:
we must use /tmp and yes it's guaranteed to be in ram. Binaries with debug are like 36mb which is more than the storage of lets say an n600 but they fit in ram just fine. So sadly we kinda have to do it this way. As for changing the ip, sure we can environment that out, but it should at least be the default for ease of use.